src/wallet/feebumper.cpp

```diff
@@ -242,7 +242,13 @@ Result CreateRateBumpTransaction(CWallet& wallet, const uint256& txid, const CCo
     if (coin_control.m_feerate) {
         // The user provided a feeRate argument.
         // We calculate this here to avoid compiler warning on the cs_wallet lock
-        const int64_t maxTxSize{CalculateMaximumSignedTxSize(*wtx.tx, &wallet, &new_coin_control).vsize};
+        // We need to make a temporary transaction with no inputs witnesses as the dummy signer expects them to be empty for some inputs
```

Comment:
which inputs?

Reply:
I think inputs for which we have set the weight using `coin_control.SetInputWeight()`, as we do with external inputs here.

Reply:
Clarified for that it's for external inputs.